src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
+    @Override
+    public List<Excavator> get() {
```

Comment:
The method `get()` should return a List of `Machine` objects. Please, change `List<Excavator>` to `List<Machine>`.